.github/workflows/release.yaml

```diff
@@ -333,3 +333,42 @@ jobs:
       base64-subjects: "${{ needs.combine_hashes.outputs.hashes }}"
       upload-assets: true # Optional: Upload to a new release
       provenance-name: kargo-cli.intoto.jsonl
+  update-homebrew-formula:
+    if: github.event_name == 'release'
+    permissions:
+      contents: write # Needed to push changes
+      pull-requests: write # Needed to create a pull request
+    runs-on: ubuntu-latest
+    needs: [ publish-image, publish-cli ]
+    steps:
+      - name: Checkout bottle Repository
+        uses: actions/checkout@v4
+        with:
+          repository: akuity/homebrew-tap
+          token: ${{ secrets.GITHUB_TOKEN }}
+          path: homebrew-tap
```

Comment:
If you omit `path`, this action will check out into `$GITHUB_WORKSPACE` and then you can skip the `cd` in your scripts. This is the conventional usage when you are dealing with only one repo.